### PR TITLE
test: add new checkpoint for apis/opts/config/runtime.go

### PR DESCRIPTION
**Pull request by scy1993**
### Ⅰ. Describe what this PR did
1. Add new checkpoint: TestRuntimeString;
2. Refactorying TestNewRuntime to make the code style consistent. Meanwhile, add a new test example in TestNewRuntime to make the coverage of NewRuntime is 100%.

### CODE REVIEW

File: apis/opts/config/runtime_test.go [hunk 1]
```diff
@@ -114,3 +154,35 @@ func TestRuntimeType(t *testing.T) {
...
+
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "get string of Runtime",
+			fields: fields{
+				values: &(map[string]types.Runtime{
```
Comment (Ace-Tang) [inline apis/opts/config/runtime_test.go]: Can you add a empty string and a nil values, like 
```
values: &(map[string]types.Runtime{}
values: &(map[string]types.Runtime{"": xxx}
```

**Comment by Ace-Tang** [comment]
@scy1993 , please sign-off you commit, like 
```
title: xxxxxx
    
Signed-off-by: name <email@sample.com>
```

File: apis/opts/config/runtime_test.go (hunk 1 above)
Comment (scy1993) [reply apis/opts/config/runtime_test.go]: Thanks for your suggestion. I have already added the two test cases you suggested.

**Comment by allencloud** [comment]
I think the CI failure is unrelated:
```
--- FAIL: TestTLSDialer (0.02s)
	syslog_test.go:193: failed to dial: x509: certificate has expired or is not yet valid
```
And this has been fixed in https://github.com/alibaba/pouch/pull/2651.

**Comment by allencloud** [comment]
LGTM, thanks a lot for your great work. Merging

**Pull request merged**